tests/test_profile.py

```diff
@@ -235,3 +235,48 @@ def test_that_links_in_the_services_page_return_200_code(self, base_url, seleniu
                 bad_urls.append(check_result)
 
         assert 0 == len(bad_urls), u'%s bad links found. ' % len(bad_urls) + ', '.join(bad_urls)
+
+    @pytest.mark.credentials
+    @pytest.mark.nondestructive
+    def test_that_user_can_view_external_accounts(self, base_url, selenium, vouched_user):
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+
+        assert settings.external_accounts.is_irc_form_present
+        assert settings.external_accounts.is_external_accounts_form_present
```

Comment:
Presence can be misleading as elements can be present but not displayed. You could add an `is_displayed` property to both `ExternalAccounts` and `Irc` which just returns `self._root_element.is_displayed()`.